src/features/course-submission/components/course-submission-form/index.tsx

```diff
@@ -0,0 +1,172 @@
+import { UploadOutlined } from '@ant-design/icons';
+import {
+  Button,
+  Form,
+  FormInstance,
+  Image,
+  InputNumber,
+  Select,
+  Switch,
+  Tabs,
+  Typography,
+  Upload,
+} from 'antd';
+import React, { useState } from 'react';
+import { Recipe } from '../../types';
+import { ChallengeJacket, ExpertJacket } from '../../styled';
+
+const CourseSubmissionForm = ({
+  form,
+  currentRecipe,
+}: {
+  form: FormInstance;
+  currentRecipe: Recipe;
+}) => {
+  const [currentJacket, setCurrentJacket] = useState('');
+  const [currentDifficulty, setCurrentDifficulty] = useState('');
+
+  const uploadProps = {
+    beforeUpload: () => {
+      return false;
+    },
+  };
+
+  const normaliseFile = (e: any) => {
+    if (Array.isArray(e)) {
+      return e[0].file;
+    }
+    return e.file;
+  };
+
+  const updateCurrentSong = (songId: string) => {
+    if (!songId) {
+      setCurrentJacket('');
+      setCurrentDifficulty('');
+      return;
+    }
+    currentRecipe.songs.every((song) => {
+      if (song.dishSong.songId === songId) {
+        setCurrentJacket(song.songDetails.image256);
+        setCurrentDifficulty(song.songDetails.difficulty);
+        return false;
+      }
+      return true;
+    });
+  };
+
+  return (
+    <>
+      {currentDifficulty &&
+        (currentDifficulty === 'Expert' ? (
+          <ExpertJacket src={`${process.env.ASSETS_URL}${currentJacket}`} />
+        ) : (
+          <ChallengeJacket src={`${process.env.ASSETS_URL}${currentJacket}`} />
+        ))}
+      {!currentDifficulty && (
+        <Image
+          src={`${process.env.ASSETS_URL}/songs/default/default.256.png`}
+        />
+      )}
+      <Form form={form} layout="vertical" style={{ textAlign: 'left' }}>
+        <Tabs
+          defaultActiveKey="0"
+          onChange={(activeKey) => {
+            updateCurrentSong(form.getFieldValue(`songId${activeKey}`));
+          }}
+        >
+          {[0, 1, 2].map((index) => {
```

Comment:
Perhaps for a later point in time - we should possibly avoid values like this. Although our dishes are 3 songs at this point, reusable code for future events would be beneficial (maybe we will end up with some event that requires 4 songs or something). Just food for thought, don't go changing it now :)